easybuild/easyconfigs/p/phyluce/phyluce-1.7.3_removal-Bio-Alphabet.patch

```diff
@@ -0,0 +1,22 @@
+removal of the Bio.Alphabet - it is not part of Biopython anymore (since v1.78)
+see also https://biopython.org/wiki/Alphabet
+author: Pavel Tomanek (INUITS)
```

Comment:
@pavelToman I made a small change here, by adding you as author of this patch (and then changing the SHA256 checksum for this patch accordingly in the easyconfig)